app/views/public_pages/faq.html.erb

```diff
@@ -37,7 +37,7 @@
             <h3><%=t("views.public_pages.faq.getting_started.missing_docs.q") %></h3>
           </a>
           <div class="accordion__content" id="a3">
-            <p><%=t("views.public_pages.faq.getting_started.missing_docs.a_html", :transcript_link => link_to("https://www.irs.gov/individuals/get-transcript", target: "_blank", rel: "noopener")) %></p>
+            <p><%=t("views.public_pages.faq.getting_started.missing_docs.a_html", :transcript_link => link_to("https://www.irs.gov/individuals/get-transcript", "https://www.irs.gov/individuals/get-transcript", target: "_blank", rel: "noopener", "data-track-click": "faq-irs-transcript", "data-track-attribute-position": "faq3")) %></p>
```

Comment:
this was a bug, was just linking to itself with the IRS url as text. since it had funky params it showed up in the site audit

Reply:
and it didn't have link tracking so i added it